service/get_supported_tokens.go

```diff
@@ -17,8 +17,9 @@ type SupportedToken struct {
 
 type SupportedTokens map[string]SupportedToken
 
-func GetSupportedTokens(ctx context.Context, filePath string) (SupportedTokens, error) {
+func LoadSupportedTokens(ctx context.Context, filePath string) (SupportedTokens, error) {
```

Comment:
Why did you have to rename? I don't think you changed the functionality otherwise, correct?

Reply:
again, its all part of future pair/symbol refactor
I dont see a point in loading the static file per each getSupportedTokens api...
i changed it to *load* because I use it in the handler, call it only once, and then reuse its result
GetSupportedTokens, API should to the same, i'll see its done sometime...